src/mantarray_desktop_app/queue_container.py

```diff
@@ -247,7 +247,7 @@ def get_data_analyzer_error_queue(
 
     # def _create_queues(self) -> None:
     #     """Create all the queues and assign to the instance variables."""
```

Comment:
Can this whole commented section be deleted?

Reply:
I suppose it can, I didn't actually see that when I was replacing everything